server/helpers/TokenHelper.js

```diff
@@ -0,0 +1,43 @@
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+import models from '../sequelize/models';
+
+const { Token } = models;
+
+dotenv.config();
+
+/**
+ * This class contains.
+ * two methods, one to help hashing password (hashPassword).
+ * and the second to retrieve hashed password.
+ */
+class TokenHelper {
+
+  /**
+   * Hashs the password.
```

Comment:
Consider updating the comment. This method does not hash the password, It decodes the token instead.